BraveWallet/Crypto/Accounts/Activity/AccountActivityView.swift

```diff
@@ -48,6 +49,12 @@ struct AccountActivityView: View {
       .foregroundColor(Color(.secondaryBraveLabel))
   }
   
+  private var assetRatios: [String: Double] {
+    activityStore.assets.reduce(into: [String: Double](), {
+      $0[$1.token.symbol.lowercased()] = Double($1.price)
+    })
+  }
```

Comment:
This will get executed for every transaction in the list with no change to the underlying value, maybe you can set it to a variable just above `ForEach` and use that to pass in each time?

Reply:
Yeah, good idea. It's also passed to `TransactionDetailsView` but we can use one variable at the start of the `body`.